apis/database/v1alpha1/types.go

```diff
@@ -70,8 +70,10 @@ type YugabyteClusterStatus struct {
 
 // A YugabyteCluster configures a Rook 'YugabyteCluster'
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="STATUS",type="string",JSONPath=".status.bindingPhase"
```

Comment:
What are your thoughts on exposing the `Ready` conditioned status?
https://github.com/crossplaneio/crossplane/blob/master/apis/stacks/v1alpha1/types.go#L41

Is it redundant or less important than binding phase (or state) for resources that support "binding"?

Reply:
https://github.com/crossplaneio/crossplane/blob/master/design/one-pager-stack-status-spec.md#fallback-option